lib/plugins/aws/package/compile/events/websockets/lib/validate.test.js

```diff
@@ -229,6 +229,51 @@ describe('#validate()', () => {
     ]);
   });
 
+  it('should add authorizer config when authorizer is specified as an object with both arn and name', () => {
+    awsCompileWebsocketsEvents.serverless.service.functions = {
```

Comment:
Let's configure this test with `runServerless` util. See: https://github.com/serverless/serverless/tree/master/test#unit-tests

Reply:
hi @medikoo I just did this. Please take a look to see if its appropriate